mx.truffleruby/mx_truffleruby.py

```diff
@@ -98,21 +98,34 @@ def deploy_binary_if_master(args):
         mx.log('The active branch is "%s". Binaries are deployed only if the active branch is "%s".' % (active_branch, primary_branch))
         return 0
 
-def ruby_run_specs(args):
-    mx.run(['bin/truffleruby', 'spec/mspec/bin/mspec', 'run', '--config', 'spec/truffle.mspec', '--format', 'specdoc', '--excl-tag', 'fails'] + args)
+def ruby_run_specs(launcher, format, args):
+    mx.run([launcher, 'spec/mspec/bin/mspec', 'run', '--config', 'spec/truffle.mspec', '--format', format, '--excl-tag', 'fails'] + args)
 
 def ruby_testdownstream(args):
     build_truffleruby()
-    ruby_run_specs(['--excl-tag', 'slow'])
+    ruby_run_specs('bin/truffleruby', 'specdoc', ['--excl-tag', 'slow'])
 
 def ruby_testdownstream_hello(args):
     build_truffleruby()
     mx.run(['bin/truffleruby', '-e', 'puts "Hello Ruby!"'])
 
+def ruby_testdownstream_aot(args):
+    if len(args) != 1:
+        mx.abort("Missing path to AOT image: mx ruby_testdownstream_aot <aot_bin>")
+
+    aot_bin = args[0]
+    ruby_run_specs(aot_bin, 'dot', [
+        '--excl-tag', 'graalvm',
+        '--excl-tag', 'aot',
+        '-t', aot_bin,
+        '-T-XX:YoungGenerationSize=2G', '-T-XX:OldGenerationSize=4G',
+        '-T-Xhome=' + os.getcwd(),
```

Comment:
I just wanted to make sure mx didn't always return the primary suite's root, since that obviously wouldn't work here. But I pushed the change and ran `mx testdownstream` and it did what we want. Thanks for the pointer.